zone/attack.cpp

```diff
@@ -43,6 +43,7 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 extern QueryServ* QServ;
 extern WorldServer worldserver;
 extern FastMath g_Math;
+extern bool Critical;
```

Comment:
Why do we have a bool as a global here?

Reply:
It is here to allow for the check below (4227) so that a double crit message is not displayed.  That said, there was a PR some time ago to display bot damage as normal practice so I will need to double check and see if this feature (RuleB Bot::DisplaySpellDamage) is still required.  The Heal version is still required as it allows for a bot owner to receive heal messages for a Bot outside of their normal group.

Reply:
Took a closer look.  The only item I see is damage shield damage.  With the DisplaySpellDamage rule it will display when a Bot is hit by a DS, to the Bot Owner.

Reply:
This hasn't been addressed.

Reply:
Removed the critical global, no longer needed